src/platform/linux/kmsgrab.cpp

```diff
@@ -583,6 +583,7 @@ class display_t : public platf::display_t {
 
     for(int y = 0; y < 4; ++y) {
       if(!fb->handles[y]) {
+        sd->fds[y] = -1;
```

Comment:
Could you please add a comment above your change, describing what this is doing?